core/src/components/select-popover/select-popover.tsx

```diff
@@ -135,11 +137,26 @@ export class SelectPopover implements ComponentInterface {
     const checked = options.filter((o) => o.checked).map((o) => o.value)[0];
 
     return (
-      <ion-radio-group value={checked}>
+      <ion-radio-group value={checked} onIonChange={(ev) => this.callOptionHandler(ev)}>
         {options.map((option) => (
           <ion-item class={getClassMap(option.cssClass)}>
             <ion-label>{option.text}</ion-label>
-            <ion-radio value={option.value} disabled={option.disabled} onClick={(ev) => this.rbClick(ev)}></ion-radio>
+            <ion-radio
+              value={option.value}
+              disabled={option.disabled}
+              legacy={true}
+              onClick={() => this.dismissParentPopover()}
+              onKeyUp={(ev) => {
+                if (ev.key === 'Enter' || ev.key === ' ') {
```

Comment:
RadioGroup only selects if the user presses `Space`. Should we align with that behavior? Also, any reason why we check `key` versus `code` in the radio group?

Reply:
Good call on aligning the interactions. Updated to only handle the `Space` key for now. 

Personally, I think the radio group implementation is probably the incorrect usage for `key` vs. `code`, while it technically accomplishes the same end result. `code` is used in environments where the physical location of the key matters, such as a game that is controlled by `WASD` keys.

The MDN docs somewhat touch on this: 
- https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code
- https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

`key` seems to be the default when that behavior is not required and the value of the key pressed is important. This can also include scenarios where modifier keys are important (using `Shift`, etc.).

Reply:
Ok makes sense. Can you make a tech debt ticket to migrate radio group from `code` to `key`?

Reply:
Created FW-3502 to track and linked this discussion in the ticket 👍